libafl_targets/src/libfuzzer/mutators.rs

```diff
@@ -351,7 +351,7 @@ where
             return result.replace(Ok(MutationResult::Skipped));
         }
         bytes.truncate(new_size);
-        core::mem::swap(input.bytes_mut(), &mut bytes);
+        input.bytes_mut().copy_from_slice(&bytes);
```

Comment:
@addisoncrump this no longer works since it's not a Vec. I believe it shouldn't matter?
Also, why don't we pass the original slice to the custom mutator directly?

Reply:
Ah, I guess because it `max_size` big we want to allocate the dude (could make sense to keep a max_size vec around in the struct, though)